lib/parser/resolve-transform.js

```diff
@@ -1,4 +1,11 @@
-// Insert our own rules here and send it to the parser on the init option
-module.exports = function resolveTransform(option) { //eslint-disable-line
+//TODO: Get the inq prompt and return the parser once again until there's no packages left to validate.
```

Comment:
This is wip, this is we're transforms are going later ( rules for init ) both parser(index.js) and this needs to change.